pages/api/Schema/reports/fourteenMonth/fourteenMonth.graphql

```diff
@@ -0,0 +1,65 @@
+extend type Query {
+  fourteenMonthReport(
+    accountListId: ID!
+    currencyType: FourteenMonthReportCurrencyType!
+  ): FourteenMonthReport!
+}
+
+enum FourteenMonthReportCurrencyType {
+  salary
+  donor
+}
+
+type FourteenMonthReport {
+  currencyType: FourteenMonthReportCurrencyType
+  currencyGroups: [FourteenMonthReportCurrencyGroup!]!
+  salaryCurrency: String!
+}
+
+type FourteenMonthReportCurrencyGroup {
+  currency: String!
+  totals: FourteenMonthReportCurrencyGroupTotals!
+  contacts: [FourteenMonthReportContact!]!
+}
+
+type FourteenMonthReportCurrencyGroupTotals {
+  year: Float!
+  months: [FourteenMonthReportCurrencyGroupTotalsMonth!]!
+  average: Float!
+  minimum: Float!
+}
+
+type FourteenMonthReportCurrencyGroupTotalsMonth {
+  month: ISO8601Date!
+  total: Float!
+}
+
+type FourteenMonthReportContact {
+  id: ID!
+  name: String!
+  total: Float!
+  average: Float!
+  minimum: Float!
+  months: [FourteenMonthReportContactMonths!]
+  accountNumbers: [String!]!
+  lateBy30Days: Boolean!
+  lateBy60Days: Boolean!
+  pledgeAmount: Float
+  pledgeCurrency: String
+  pledgeFrequency: String
+  status: String
+}
+
+type FourteenMonthReportContactMonths {
+  month: ISO8601Date!
+  total: Float!
+  salaryCurrencyTotal: Float!
+  donations: [FourteenMonthReportContactDonation!]!
+}
+
+type FourteenMonthReportContactDonation {
```

Comment:
Why this new donation type instead of the already existing `Donation`?

Reply:
Cuz I was making all new stuff. I'll look into reusing that... Good point.

Reply:
I don't think I have all the data to fill all the required fields of it ☹️ 
<img width="601" alt="Screen Shot 2021-06-14 at 12 22 54 PM" src="https://user-images.githubusercontent.com/756501/121947862-67847380-cd0b-11eb-9b02-f264829a9915.png">